rest/class-boldgrid-backup-rest-setting.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * File: class-boldgrid-backup-rest-setting.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      X.X.X
```

Comment:
you're using X.X.X in this file instead of SINCEVERSION ..... Just putting this here for ya so these don't get missed when you do your search and replace later

Reply:
Thanks for the heads up https://github.com/BoldGrid/boldgrid-backup/commit/56aad88cc9172596a6d3868908f807a508f28a17